hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -158,11 +158,12 @@ public function upgradeBundle() {
     CRM_Core_DAO::executeQuery("DROP TABLE IF EXISTS civicrm_hrhours_location");
     CRM_Core_DAO::executeQuery("
         CREATE TABLE IF NOT EXISTS `civicrm_hrhours_location` (
-        `id` int(10) unsigned NOT NULL,
+          `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
You're right!  What I found is it's impossible to add records to this table, so I don't think it's ever worked.  Good point.